web/src/locales/zh-CN/component.ts

```diff
@@ -21,19 +21,24 @@ export default {
   'component.global.search': '查询',
   'component.global.reset': '重置',
   'component.global.confirm': '确认',
+  'component.global.format': '格式',
+  'component.global.document': '文档',
+  'component.global.enable': '使能够',
+  'component.global.scope': '范围',
+  'component.global.data.editor': '资料编辑器',
   'component.global.delete': '删除',
   'component.global.cancel': '取消',
   'component.global.submit': '提交',
   'component.global.create': '创建',
   'component.global.createWithEditor': '使用编辑器创建',
   'component.global.add': '新建',
   'component.global.save': '保存',
-  'component.global.edit': '编辑',
+  'component.global.edit': '配置',
```

Comment:
actually, it's more clear to use `编辑` 😂

```suggestion
  'component.global.edit': '编辑',
```